File: iTSoft.CRM.UI/src/app/_models/employeeSelectListModel.ts
import { ListModel } from './listmodel';

export class EmployeeSelectListModel
{
    Roles: ListModel[];
    Designations: ListModel[];
    Departments : ListModel[];
    Emails : ListModel[];
   
    constructor()
    {
        this.Roles=[];
        this.Designations=[];
        this.Departments=[];
        this.Emails = [];
    }
}

File: iTSoft.CRM.Web_Old/iTSoft.CRM.Domain/Models/ViewModel/EmployeeSelectListModel.cs
using iTSoft.CRM.Data.Entity;
using System;
using System.Collections.Generic;
using System.Text;

namespace iTSoft.CRM.Domain.Models.ViewModel
{
   public class EmployeeSelectListModel
    {
        public List<ListModel> Roles { get; set; }
        public List<ListModel> Designations { get; set; }
        public List<ListModel> Departments { get; set; }
        public List<ListModel> Emails { get; set; }
    }
}
